sample/team-props/checkstyle-suppressions.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+  "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+  "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+
+  <suppress checks=".*" files=".*\.kt" />
```

Comment:
wouldn't be enough to have this setting in the buildscript using the `checkstyle.exclude()`?

Reply:
indeed. and it's another plugin functionality being showcased.